ui/src/builder/BuilderCopyText.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<span class="tooltip">
+		<span class="tooltiptext">{{ copied }}</span>
+		<span class="copyText" @click="copyText">
+			<slot></slot>
+		</span>
+	</span>
+</template>
+
+<script setup lang="ts">
+import { ref } from "vue";
+const copied = ref("Copy to clipboard");
+
+const copyText = (e) => {
+	const copyText = e.target.textContent;
+	navigator.clipboard.writeText(copyText);
+	copied.value = "Copied!";
+	setTimeout(() => {
+		copied.value = "Copy to clipboard";
+	}, 1000);
+};
+</script>
+
+<style scoped>
+.tooltip {
+	position: relative;
+	display: inline-block;
+}
+
+.tooltip .tooltiptext {
+	visibility: hidden;
+	background-color: var(--builderActionOngoingColor);
+	color: #fff;
+	text-align: center;
+	border-radius: 20px;
+	padding: 5px 10px;
```

Comment:
I try quite hard to use multiples of `4px` in margins, paddings, border-radiuses, widths, etc e.g. `4px 12px`. Reason is maintaining a grid and hopefully one day migrating to a more sophisticated way of managing spacing. It's less of an issue in Builder, but still I'd prefer it.